package/com.unity.formats.usd/Runtime/Scripts/IO/Scene/ImportHelpers.cs

```diff
@@ -59,7 +59,7 @@ public static GameObject ImportSceneAsGameObject(Scene scene, GameObject parent
         }
 
 #if UNITY_EDITOR
-        public static string ImportAsPrefab(Scene scene)
+        public static string ImportAsPrefab(Scene scene, string prefabPath)
```

Comment:
If this (and ImportAsTimelineClip below) are public APIs that users may already be using, I think we should give this new parameter a default of null to not give them compile errors when they upgrade

Reply:
I updated the API